### PR TITLE
std housekeeping

### PR DESCRIPTION
**Pull request by ominitay**
Spotted a few TODOs, thought it best to act on them now 0.9.0 is released.

### CODE REVIEW

**Comment by ominitay** [comment]
Seems that uncovered some broken code previously undetected thanks to the compileErrors

**Comment by ominitay** [comment]
Rebased. This should be merged or rejected before it becomes stale.

File: lib/std/zig/parser_test.zig [hunk 1]
```diff
@@ -72,21 +72,6 @@ test "zig fmt: rewrite callconv(.Inline) to the inline keyword" {
     );
 }
 
-// TODO Remove this after zig 0.9.0 is released.
-test "zig fmt: rewrite @byteOffsetOf to @offsetOf" {
```
Comment (Vexu) [inline lib/std/zig/parser_test.zig:76]: The actual rewriting this is testing should also be removed.

Comment (ominitay) [reply lib/std/zig/parser_test.zig:76]: Ah I see, will sort that out in a sec.

Comment (ominitay) [reply lib/std/zig/parser_test.zig:76]: Should be ok now :)

**Pull request merged**